extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/test/java/io/quarkus/resteasy/reactive/server/test/resource/basic/ResourceLocatorTest.java

```diff
@@ -61,6 +61,7 @@ public static void close() {
 
     @RegisterExtension
     static QuarkusUnitTest testExtension = new QuarkusUnitTest()
+            .overrideConfigKey("quarkus.http.root-path", "/app")
```

Comment:
A bit late to the party, but I worry that this will now have one less test where the path is empty. Perhaps we should have duplicated this test instead? One with path, one without?

Reply:
We actually already have a version of the test in the independent project, so we should be fine